DataRepo/templates/navbar.html

```diff
@@ -45,6 +38,11 @@
                     Feedback
                 </a>
             </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="https://drive.google.com/drive/folders/16_KYcHbX_dwoQuYgFEVBRHoyl9jlilUg?usp=sharing" id="dropdown-help" role="button" data-bs-toggle="dropdown" aria-expanded="false">
```

Comment:
NB.

Not sure what the param does, but it works without it...
```suggestion
                <a class="nav-link dropdown-toggle" href="https://drive.google.com/drive/folders/16_KYcHbX_dwoQuYgFEVBRHoyl9jlilUg" id="dropdown-help" role="button" data-bs-toggle="dropdown" aria-expanded="false">
```

NB. Seems a bit unexpected for the text "help" to link to a GD folder. Might be better to either change the link text or create a help page where the folder is linked?